### DISCUSSION TITLE
Help Email notification not working in eXo Platform community Edition

### DISCUSSION THREAD

**Discussion by Kevincioo** [exoplatform/exo-community#17] [Q&A]
Hello 

I successfully installed the eXo community edition however when I want to send an invitation to the user the email is not working also in the apps I install on ubuntu 20.04.6 LTS

**Comment by Kevincioo** [comment]
![image](https://user-images.githubusercontent.com/67741605/225765426-1d3e2c60-241e-4d56-9fd4-150e0a130936.png)

**Comment by rdenarie** [comment]
Hello @Kevincioo 

In fact, the default configuration of the platform to send mail is to use localhost:25
And it seems that there is no mail server on this address/port on your machine.

You have 2 options here : 
- use a docker container that will collect all mails. No mails will be really send but all will be catch by the application. You can use something like MailHog. It can be launched using this docker command : 
`docker run -d -p 1025:1025 -p 8025:8025 mailhog/mailhog`
Simply add the following property 
`EXO_MAIL_SMTP_PORT=1025`
in the docker compose for the 'exo' component. And then you can access to `http://localhost:8025` to see received mails. All mails will be visible, even if using fake mail address. It is quite usefull for tests purpose.
- use a real mail server (like the Gmail one, you need to have a Gmail account. Other  providers are available, choose the more suitable for you). For that, you can configure different mail properties in docker compose file : 
https://hub.docker.com/r/exoplatform/exo-community#mail (See the 'Mail' part).
For gmail smtp options, you can look at [this](https://kinsta.com/blog/gmail-smtp-server/#how-to-find-the-smtp-server-for-gmail) for example

**Reply by Kevincioo** [reply]
Thank you so much @rdenarie  also may I know how I can put SSL on my eXo platform?

**Reply by murthdaddy** [reply]
Does the gmail smtp option still work?  
I've been attempting to configure it but have been unsuccessful.  Read that over the past year they tighten up third-party/App access to their smtp, so wondering if that is the issue or I'm just off a bit on the config.

Here is my config in exo.properties file:
<img width="521" alt="image" src="https://github.com/exoplatform/exo-community/assets/168120613/df000b20-0c52-484e-8c69-a53d27c65616">

**Reply by rdenarie** [reply]
Hello. 
It should work. Please check if you have Multifactor Authentication enabled on you gmail account. In this case you should create a application password ([example](https://www.123formbuilder.com/docs/setting-up-smtp-with-2-step-verification-on/))
Can you open a dedicated subject on https://community.exoplatform.com ?
You will have more visibility and anwsers.

**Reply by murthdaddy** [reply]
Thank you @rdenarie !  Your example resource was great and it helped me fix things with my gmail account to get it working with gmail SMTP...in particular, enabling MFA, creating an App password, & disabling IMAP.

**Comment by rdenarie** [comment]
Hello @Kevincioo 
To activate SSL, you can follow this documentation : 
https://docs.exoplatform.org/administration/security.html#enabling-https

Follow the part "Use a reverse proxy", and update the nginx configuration you get during the eXo community installation
